layers/spell-checking/config.el

```diff
@@ -10,11 +10,14 @@
 ;;
 ;;; License: GPLv3
 
-;; Variables
-
 ;; Command Prefixes
 
 (spacemacs/declare-prefix "S" "spelling")
 
+;; Variables
+
 (defvar spell-checking-enable-by-default t
   "Enable spell checking by default.")
+
+(defvar spell-checking-auto-dictionary t
```

Comment:
This variable is not needed, the user can exclude a package with `dotspacemacs-excluded-packages`.